### DISCUSSION TITLE
Option for uninstall without dustbin

### DISCUSSION THREAD

**Discussion by Laserman49** [Klocman/Bulk-Crap-Uninstaller#454] [Ideas]
Can we have an option to Uninstall with nothing in the dustbin please ?

**Comment by Klocman** [comment]
What's a dustbin?

**Reply by ghost** [reply]
If he's talking about the Windows recycle bin, that's it, if not I don't know but dustbin is a British word for trash or garbage

**Comment by Klocman** [comment]
You can turn off using recycle bin when removing junk in BCU settings.
